example/leaflet.spin.min.js

```diff
@@ -1,2 +1,2 @@
-!function(n,i){"function"==typeof define&&define.amd?define(["leaflet"],function(i){n(i)}):"object"==typeof exports?module.exports=function(i){return void 0===i&&(i=require("leaflet")),n(i),i}:"undefined"!=typeof i&&i.L&&n(i.L)}(function(n){var i={spin:function(n,i){n?(this._spinner||(this._spinner=new Spinner(i).spin(this._container),this._spinning=0),this._spinning++):(this._spinning--,this._spinning<=0&&this._spinner&&(this._spinner.stop(),this._spinner=null))}},t=function(){this.on("layeradd",function(n){n.layer.loading&&this.spin(!0),"function"==typeof n.layer.on&&(n.layer.on("data:loading",function(){this.spin(!0)},this),n.layer.on("data:loaded",function(){this.spin(!1)},this))},this),this.on("layerremove",function(n){n.layer.loading&&this.spin(!1),"function"==typeof n.layer.on&&(n.layer.off("data:loaded"),n.layer.off("data:loading"))},this)}
+!function(n,i){"function"==typeof define&&define.amd?define(["leaflet"],function(i){n(i)}):"object"==typeof exports?module.exports=function(i){return void 0===i&&(i=require("leaflet")),n(i),i}:void 0!==i&&i.L&&n(i.L)}(function(n){var i={spin:function(n,i){n?(this._spinner||(this._spinner=new Spinner(i).spin(this._container),this._spinning=0),this._spinning++):--this._spinning<=0&&this._spinner&&(this._spinner.stop(),this._spinner=null)}},t=function(){this.on("layeradd",function(n){n.layer.loading&&this.spin(!0),"function"==typeof n.layer.on&&(n.layer.on("data:loading",function(){this.spin(!0)},this),n.layer.on("data:loaded",function(){this.spin(!1)},this))},this),this.on("layerremove",function(n){n.layer.loading&&this.spin(!1),"function"==typeof n.layer.on&&(n.layer.off("data:loaded"),n.layer.off("data:loading"))},this)}
 n.Map.include(i),n.Map.addInitHook(t)},window)
```

Comment:
Yo do not need to include `leaflet.spin.min.js` twice in sources. A symlink in `example`, linking the file from parent directory should make it.